runtime/development/src/lib.rs

```diff
@@ -937,28 +997,32 @@ parameter_types! {
 }
 
 impl pallet_treasury::Config for Runtime {
-	// either democracy or 75% of council votes
 	type ApproveOrigin = EnsureRootOr<
 		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 4>,
 	>;
 	type Burn = Burn;
-	// we burn and dont handle the unbalance
 	type BurnDestination = ();
 	type Currency = Tokens;
 	type MaxApprovals = MaxApprovals;
-	// slashed amount goes to treasury account
 	type OnSlash = Treasury;
 	type PalletId = TreasuryPalletId;
 	type ProposalBond = ProposalBond;
 	type ProposalBondMaximum = ProposalBondMaximum;
 	type ProposalBondMinimum = ProposalBondMinimum;
-	// either democracy or more than 50% council votes
 	type RejectOrigin = EnsureRootOr<HalfOfCouncil>;
 	type RuntimeEvent = RuntimeEvent;
 	type SpendFunds = ();
 	type SpendOrigin = frame_support::traits::NeverEnsureOrigin<Balance>;
 	type SpendPeriod = SpendPeriod;
 	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
+	type AssetKind = ();
+	type Beneficiary = Self::AccountId;
+	type BeneficiaryLookup = IdentityLookup<Self::Beneficiary>;
+	type Paymaster = PayFromAccount<Balances, TreasuryAccount>;
+	type BalanceConverter = UnityAssetBalanceConversion;
+	type PayoutPeriod = ConstU32<10>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = ();
```

Comment:
pallet-treasury has change a lot allowing to transfer funds from the treasury [ref](https://github.com/paritytech/polkadot-sdk/pull/1333). 

By now, I've set this simple as if we are not using the new capabilities. I think if we want this, we should configure it later in a new PR after this upgrade to avoid mixing things up. Note that a correct configuration requires the `pallet_asset_rate` pallet to be added, so it's not trivial.